.github/workflows/build-node-and-runtime.yml

```diff
@@ -44,15 +44,37 @@ jobs:
       - name: Upload binary
```

Comment:
just for consistency: `Upload release binary`

Reply:
Changed.